server/routes/offenceRoutes.ts

```diff
@@ -1965,7 +1972,7 @@ export default class OffenceRoutes extends BaseRoutes {
       addOrEditCourtCase,
       addOrEditCourtAppearance,
     } = req.params
-
+    const submitToEditOffence = req.query
```

Comment:
```suggestion
    const { submitToEditOffence } = req.query
```
destructure the query object so that only submitToEditOffence is considered